docs/koji.md

```diff
@@ -23,6 +23,7 @@ For atomic-reactor container image builds the `image` type is used, and so type-
 Data which is placed here includes:
 
 - `build.extra.image.autorebuild` (boolean): true if this build was triggered automatically; false otherwise
+- `build.extra.image.help` (string or null): filename if this build has a markdown help converted; null otherwise
```

Comment:
I feel like we need to make it clear that it is the path to the Markdown file in the git repository (rather than to the converted file in the built image).